### DISCUSSION TITLE
Filters for Device and Entity Lists

### DISCUSSION THREAD

**Discussion by dougiteixeira** [home-assistant/frontend#11055] [Dashboard]
Include the possibility to filter the list of devices by manufacturer, model, integration or area and also the list of entities by integration or area.

**Comment by Rick-Git** [comment]
Was thinking the same this weekend. My current HA-instance isn't running much but will explode within a year (moving to a new home, woohoo). What really helps is the possibility to filter entities by ID: light, switch, sensor, binary_sensor, etc with the /config/entities page!
